polaris/utils/misc.py

```diff
@@ -16,3 +18,19 @@ def slugify(sluggable: SlugCompatibleStringType) -> SlugStringType:
     Converts a slug-compatible string to a slug.
     """
     return sluggable.lower().replace("_", "-").strip("-")
+
+
+def convert_lists_to_arrays(predictions: ListOrArrayType | dict) -> np.array | dict:
```

Comment:
**Fix**
The rare moment where `Union` is required for Python to make sense of the type signature.

```suggestion
def convert_lists_to_arrays(predictions: Union[ListOrArrayType, dict]) -> np.array | dict:
```